src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: FC<Props> = ({ person }) => {
+  const { peopleSlug } = useParams();
+
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+  } = person;
+
+  const currentRow = slug === peopleSlug;
+  // const parentsName = (
+  //   (fatherName || motherName) ? peopleSlug?.split('-')
+  //     .slice(0, 2)
+  //     .join(' ') === person.name.toLowerCase() : false
+  // );
+
+  // const handleClickParents = () => {
+  //   setParentsName()
+  // }
+
+  const woman = person.sex === 'f';
+  const isMother = motherName || '-';
+  const isFather = fatherName || '-';
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': currentRow,
+      })}
+
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={classNames({
+            'has-text-danger': woman,
+          })}
+        >
+          {name}
+        </Link>
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        {person.mother ? (
+          <Link
+            to={`/people/${person.mother?.slug}`}
+            className={classNames({
+              'has-text-danger': isMother,
```

Comment:
The 'isMother' variable is a string, so using it to conditionally apply a class could be misleading. Consider using a more appropriate condition for the 'has-text-danger' class.

Reply:
Not fixed

Reply:
I have changed the variable isMother to isWoman=person.sex === 'f'. That variable has boolean value.